explorer/app/components/Header/DrawerContent.tsx

```diff
@@ -0,0 +1,38 @@
+import { Typography } from "@mui/material";
+import React from "react";
+import { LogoProps } from "../Logo/Logo";
+import { NavLinksProps, NavLinks } from "../NavLinks/NavLinks";
+import { SocialLinksProps, SocialLinks } from "../SocialLinks/SocialLinks";
+import {
+  DrawerContainer,
+  SloganDrawerContainer,
+  SocialDrawerContainer,
+} from "./Header.styles";
+
+interface DrawerContentProps {
+  logo: LogoProps;
+  navLinks: NavLinksProps;
+  socialLinks: SocialLinksProps;
+}
+
+export const DrawerContent = ({
+  logo,
+  navLinks,
+  socialLinks,
+}: DrawerContentProps) => {
+  return (
+    <DrawerContainer>
+      {logo.slogan && (
+        <SloganDrawerContainer>
```

Comment:
This component is looking great! We can probably remove the need for the slogan wrapper container and utilise a combination of the Mui `sx` [system](https://mui.com/system/properties/) with Typography props and and achieve the same outcome with:

```
<Typography component="div" color="colorInk" py={2} px={6} variant="textBody400">{logo.slogan}</Typography>
```

Reply:
Cool!! I just changed the `py={2}` to `pb={2}` since we're keeping the padding-top for the `DrawerContainer`. Is that ok?